corehq/apps/case_search/utils.py

```diff
@@ -70,7 +70,7 @@ def _assemble_optional_search_params(self):
     def _validate_param_value(self, key, value):
         if isinstance(value, list):
             raise CaseFilterError(
-                _("Multiple xpath queries for a query param is not supported"),
+                _("Multiple values for this param is not supported"),
```

Comment:
nit: I would remove 'this' and keep it non-specific. I don't think the specific param is even passed in the response which should probably change: https://github.com/dimagi/commcare-hq/blob/d552766151273116af6715633ea888bcd8da021b/corehq/apps/ota/views.py#L120